examples/js/plugins/FeatureToolTip.js

```diff
@@ -81,9 +85,16 @@ const FeatureToolTip = (function _() {
         for (let p = 0; p < features.length; p++) {
             feature = features[p];
             geometry = feature.geometry;
-            style = (geometry.properties && geometry.properties.style) || feature.style || layer.style;
-            const context = { globals: {}, properties: getGeometryProperties(geometry) };
-            style = style.drawingStylefromContext(context);
+
+            const globals = {
+                fill: true,
+                stroke: true,
+                point: true,
+                icon: true,
+            };
+            const context = { globals, specifics: { properties: getGeometryProperties(geometry), type: getFeatureType(feature) } };
+
+            style = itowns.Style.merge(layer.style, feature.style, context).drawingStylefromContext(context);
 
```

Comment:
Yes, This is something that indeed can be improved by merging this 2 functions but It will be better be done when we move all Style to Layer. as for the moment we still need 2 styles interface for the layer/user style and the data style.